go.mod

```diff
@@ -7,8 +7,10 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/gorilla/websocket v1.4.1
 	github.com/mdlayher/raw v0.0.0-20191009151244-50f2db8cc065
+	github.com/otiai10/copy v1.0.1
 	github.com/pin/tftp v2.1.0+incompatible
 	github.com/satori/go.uuid v1.2.0
+	golang.org/x/crypto v0.0.0-20191122220453-ac88ee75c92c
```

Comment:
run `go mod tidy` to get rid of unused packages

Reply:
Committed.